src/main/java/dk/cngroup/lentils/entity/Hint.java

```diff
@@ -13,9 +13,9 @@
     @Column(name = "hint_id")
     private Long hintId;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Tady asi nebude `CascadeType.All`. Znamenalo by to napriklad to, ze pokud smazes Hint, tak se smaze i Sifra, coz asi nechceme. Muzes to Jito prosim overit, jestli to tak skutecne funguje prosim?